.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+## Motivation and Context
+
+<!--
+  Why is this change required? What problem does it solve?
+  This can be omitted if a linked issue is provided 
+-->
+
+## Description
+
+<!-- Describe your changes -->
+
+## Test Plan
+
+<!-- Please describe in detail how you tested your changes. -->
+
+## Screenshots (if appropriate):
+
+<!-- 
+  This allows reviewers to begin reviewing your work 
+  without checking out your branch locally
+-->
+
```

Comment:
```suggestion
## Reviewers

Tag any reviewers who have context on this PR, or are familiar with this part of the codebase.
```

Reply:
I think this section makes sense as a normal contributor cannot request reviews, added with https://github.com/coordinape/coordinape/pull/913/commits/1c4a80581366929046bb6058d471c8105977ce6b